pkg/controller/network/securitygroup/managed_test.go

```diff
@@ -0,0 +1,470 @@
+package SecurityGroup
+
+import (
```

Comment:
these imports will fail linting https://github.com/crossplane/provider-azure/blob/b55e3a0dabc627726d2f5275d80f662650932d45/.golangci.yml#L38

Reply:
Sorry I am unable to understand this statement , could you please share more details what changes are expected.

Reply:
We require imports to be structured in groups separated by a blank line in the following order:
1. stdlib packages
2. external packages (i.e. not from crossplane org)
3. other crossplane org packages (i.e. `crossplane-runtime`)
4. provider-azure packages (i.e. from within this repo)

Reply:
First running:
`goimports -local github.com/crossplane/provider-azure  -w pkg/controller/network/securitygroup/managed_test.go pkg/controller/network/securitygroup/managed_test.go`, 
and then running:
`goimports -local github.com/crossplane/crossplane-runtime  -w pkg/controller/network/securitygroup/managed_test.go pkg/controller/network/securitygroup/managed_test.go`, 
should put them into shape.